bee-message/src/payload/milestone/essence.rs

```diff
@@ -134,6 +152,9 @@ impl Packable for MilestoneEssence {
             verify_pow_scores(index, next_pow_score, next_pow_score_milestone_index).map_err(UnpackError::Packable)?;
         }
 
+        let metadata = BoxedSlicePrefix::<u8, MilestoneMetadataLength>::unpack::<_, VERIFY>(unpacker)
+            .map_packable_err(|err| Error::InvalidMilestoneMetadataLength(err.into_prefix_err().into()))?;
```

Comment:
```suggestion
            .map_packable_err(|e| Error::InvalidMilestoneMetadataLength(e.into_prefix_err().into()))?;
```
Sorry for being this nit-picky but down below you also use `e` or at least `_e`, so it's just for crate consistency I guess.